web/docs/guides/database/password-reset.mdx

```diff
@@ -0,0 +1,57 @@
+---
+id: password-reset
+title: "Reset Database Password"
+description: How to change your PostgreSQL database password.
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+### How to change your PostgreSQL database password.
+
+## Description
+
+Your PostgreSQL database is the core of your Supabase project, so it's important to make sure it has a strong, secure password at all times.  Changing the password is easy -- just make sure you choose a password that:
+
+* is very secure
+* does not contain of the following symbols: **: / ? # ] [ @ ! $ & ' ( ) * + , ; =** 
+
+These special symbols can conflict with connection string URLs or cause other problems, so it's best to avoid them.  You can still make a secure password using a longer string or a randomized set of upper and lowercase letters and digits.
+
+## Environment
+* Any PostgreSQL database
+* Any Supabase project
+
+## Steps
+
+Issue the following statement in a SQL editor:
+
+From the [Supabase Dashboard](https://supabase.io): 
+- Home 
+- Select a Project 
+- SQL 
+- New Query
+
+<Tabs
+defaultValue="SQL"
+values={[
+  {label: 'SQL', value: 'SQL'},
+]}>
+
+<TabItem value="SQL">
+
+```sql
+ALTER USER postgres WITH PASSWORD 'new_password';
```

Comment:
how do we update our middleware db with this new password? Or do we assume that they can go out of sync?

Reply:
It's OK to get out of sync - we don't use the postgres user for super_admin tasks